cli.js

```diff
@@ -55,6 +55,12 @@ const cli = meow(`
 
 (async () => {
 	try {
+		const {rename} = cli.flags;
+		const renameReg = /{{(\s*)?basename(\s*)?}}/;
```

Comment:
I would simply just support exactly `{{basename}}`. No regex needed.

Reply:
@sindresorhus  Use regular expressions to prevent spaces from causing no matches.

Reply:
I understand why regex is used. I'm just saying I don't think it's needed. Users should not assume that spaces are allowed. Using regex may seem nice, but regex comes with the risk of introducing ReDoS. I try to avoid regexes whenever possible.

Reply:
@sindresorhus regular expressions have been removed.